website/docs/artist_details_panel.md

```diff
@@ -0,0 +1,40 @@
+---
+id: artist_details_panel
+title: The Task Details Panel
+sidebar_label: Details Panel
+---
+
+:::info
+The details panels received updates in server version `1.3.0`.
+:::
+
+## Overview 
+When using the AYON production tracker, you'll encounter the details panel frequently. It's essential for team interactions.
+As the name suggests, the details panel is packed with information about tasks, serving as a communication tool and offering useful actions.
+
+
+## Panel Content
+
+![](assets/details_panel/details_panel_overview.png)
+
+- **Folder Path**: Displays the folder path of the current task.
+- **Name & Thumbnail**: Shows the name and thumbnail of the task.
+:::tip Update Thumbnail
+You can update the thumbnail by right-clicking the thumbnail and selecting Upload new thumbnail.
+:::
+- **Tags**: View and add tags.
+- **Status**: View and update status.
+- **Web Actions**: Access available web actions to launch applications directly via the AYON web server.
```

Comment:
```suggestion
- **Web Actions**: Access available web actions, e.g. applications, directly via the AYON web server.
```

Reply:
Thank you. 
https://github.com/ynput/ayon-documentation/pull/287/commits/069be35feeac7fabad542741b1bfa6d7643d8195